snap/squashfs/squashfs.go

```diff
@@ -298,28 +298,33 @@ func (s *Snap) ListDir(dirPath string) ([]string, error) {
 }
 
 // Build builds the snap.
-func (s *Snap) Build(buildDir, snapType string) error {
+func (s *Snap) Build(sourceDir, snapType string, excludeFiles ...string) error {
 	fullSnapPath, err := filepath.Abs(s.path)
 	if err != nil {
 		return err
 	}
-
-	args := []string{
+	cmd, err := osutilCommandFromCore("/usr/bin/mksquashfs")
+	if err != nil {
+		cmd = exec.Command("mksquashfs")
+	}
+	cmd.Args = append(cmd.Args,
 		".", fullSnapPath,
 		"-noappend",
 		"-comp", "xz",
 		"-no-fragments",
 		"-no-progress",
+	)
+	if len(excludeFiles) > 0 {
+		cmd.Args = append(cmd.Args, "-wildcards")
+		for _, excludeFile := range excludeFiles {
+			cmd.Args = append(cmd.Args, "-ef", excludeFile)
```

Comment:
Just wondering, does it work when you pass more than one exclude file?

Reply:
yep!

maybe i should add a test for it though :-)